CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Cobra
+
+Thank you so much for contributing to Cobra. We appreciate your time and help.
+Here are some guidelines to help you get started.
+
+## Code of Conduct
+
+Be kind and respectful to the members of the community. Take time to educate
+others who are seeking help. Harassment of any kind will not be tolerated.
```

Comment:
Maybe we can generate a `CODE_OF_CONDUCT.md` file from Github. I believe the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md)  may work well for our project.

Reply:
+1 for adding the Covenant Code of Conduct, especially if it's a recommended code by Github